rbhl/static/css/rbhl.scss

```diff
@@ -28,8 +28,12 @@ $twocolwidth: 16.6666666667%;
 // doing this means the page does not
 // jump when there is a scroll bar.
 // Forcing a scroll bar looks good.
-html {
+html, body{
     overflow-y: scroll;
+    height: 100%;
+    @media print{
+        overflow: visible;
```

Comment:
don't show the scroll bar in the print view (scroll bar is now always there to stop page shift)